libpod/define/container_inspect.go

```diff
@@ -529,6 +533,10 @@ type InspectContainerHostConfig struct {
 	IOMaximumBandwidth uint64 `json:"IOMaximumBandwidth"`
 	// CgroupConf is the configuration for cgroup v2.
 	CgroupConf map[string]string `json:"CgroupConf"`
+	// Timeout is time before container is killed by conmon
+	Timeout uint `json:"Timeout"`
```

Comment:
These should be in config or hostconfig, not both

Reply:
Oops from a previous version.